templates/base/.gitignore

```diff
@@ -12,6 +12,9 @@
 /.next
 /out/
 
+# storybook output folder
+/storybook-output
```

Comment:
@vinodloha - Is that working fine on your end? I am now able to see untracked files from this folder.

Reply:
It has to work same as /build below.